ios/FluentUI/Avatar/Persona.swift

```diff
@@ -42,6 +42,10 @@ public protocol Persona {
 
     /// The subtitle value for the persona.
     var subtitle: String { get }
+
+    /// Loads the image asynchronously. Implement this instead of use ``image``  property if it needs to be fetched asynchronously. Image caching should be handled in the implementation for optimial efficiency.
+    /// - Parameter completion: The completion block that returns the image
+    @objc optional func loadImageAsync(completion: @escaping (UIImage?) -> Void)
```

Comment:
also wroth adding a test case for  demo controller of this usage.

Reply:
I thought about it, but will need some design idea on this.
I was thinking have a toggle to switching between using the static image vs loading async, but unsure on how to present the toggle. Alternatively, we can do both at the same time, use the static image first, then do time delay to simulate background task, and set the same image after some delay?

Reply:
One idea that came into my mind is quite similar to what you suggest, @lingzlu:
That idea is basically about adding a toggle switch at the top of the PeoplePicker demo controller. Something like  "Delay load images" or "Load persona images asynchronously".
Then if that toggle is on, the images are set asynchronously after a timeout (of maybe 1 second?). If not, they're just set directly as the current state of things.

![Screen Shot 2022-02-07 at 9 31 29 AM](https://user-images.githubusercontent.com/68076145/152840871-903a0cf7-0475-47c1-a4ad-4ebb190be49f.png)

Reply:
@rdeassis thanks for sharing the idea! I can mock something like you mentioned:

> "Load persona images asynchronously     [Toggle] ".

Another idea that just came to mind is that given there are two demo views for persona: `PeoplePicker` and `PersonaListView`, I can update `PeoplePickerDemoController` to use a new list of `samplePersonas` objects that are subclass of `PersonaData` with `fetchImage` implementation. From the scenario stand-point, it also make sense for people picker to load image asynchronously. It's also nice that we don't have to worry about UI mock up. Let me know what you think. 

Another request is that we (Planner iOS team) would like to have the change available as soon as possible, are you and @harrieshin okay with me doing a follow up change  for the demo?

Reply:
Hey @lingzlu my personal opinion is that the toggle showcases the ability to set avatar images asynchronously better because it explicitly calls out that feature that can be turned on/off in the demo controller.

I'm holding on the [release of 0.4.4](https://github.com/microsoft/fluentui-apple/pull/893) so that this change can get in that version and be released this week.

Regarding the deferral of the Demo controller changes into another PR:
What would be the timeline for that followup change on the demo controller?
The reason I ask is because it seems to be a fairly simple change to make to the demo controller and ideally we'd like to have it all merged together.

Reply:
Hey @lingzlu, just a heads up that we needed to release the version 0.4.4.
Your change will be included in version 0.4.5 and we can expedite it once you merge this PR.
Do you think you can include the demo controller changes in this PR still?

Reply:
hey @rdeassis, sorry, it has been a busy week for me. I probably won't get to it until early next week (likely next Monday). I know the change isn't big, the request for the deferral is out of the concern for the review process. What is your target date for the next release?